### DISCUSSION TITLE
Spleeter and M1

### DISCUSSION THREAD

**Discussion by scorbobby** [community/community#69909] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

MySpleeter doesn't work. Is it because it's not compatible with Mac M1?

**Comment by thevindu-w** [comment]
They have mentioned that it is not yet supported on M1 and M2.
Reference: [https://github.com/kyab/MySpleeter#m1m2-mac-support](https://github.com/kyab/MySpleeter#m1m2-mac-support)

**Reply by scorbobby** [reply]
Thank you, Thevindu. I wanted to use it today to isolate a bass part I'm trying to learn.
Robert

**Comment by scorbobby** [comment]
Thank you, Thevindu. I wanted to use it today to isolate a bass part I'm
trying to learn.
Robert
